libs/langchain/package.json

```diff
@@ -32,6 +32,7 @@
     "format:check": "prettier --config .prettierrc --check \"src\""
   },
   "devDependencies": {
+    "@anthropic-ai/sdk": "^0.56.0",
```

Comment:
we need to remove this once we figure out where the anthropic file helpers should go; left here for now so people can play with the code